<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyDialog</class>
 <widget class="QDialog" name="ModifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="SexEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="MerryEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="WhereEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>56</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>民族:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>政治面貌:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>员工号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>办公电话:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>到岗日期:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>备注:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="BackgroundEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PersonNumberEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StateEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>职务:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SituationEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>56</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>出生日期:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>文化程度:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="OtherEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>填表人: admin</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>填表日期:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>2010-01-01</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>工作岗位:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PeriodEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DesignButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="NameEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="BrithdayEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="CultureEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>员工状态:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NationEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>身份证号:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="ReachEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlaceEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>户口所在地:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkNumEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>56</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>性别:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>婚姻状况:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>籍贯:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="TelEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkTelEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PostEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>合同有效期:</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>79</width>
      <height>109</height>
     </rect>
    </property>
    <widget class="QLabel" name="photolabel_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="photolabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>-7</x>
     <y>0</y>
     <width>601</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/background_meitu_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_23</zorder>
  <zorder>scrollArea</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>BackgroundEdit</zorder>
  <zorder>PersonNumberEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>StateEdit</zorder>
  <zorder>label_11</zorder>
  <zorder>SituationEdit</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>OtherEdit</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>label_10</zorder>
  <zorder>PeriodEdit</zorder>
  <zorder>DesignButton</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>SubmitButton</zorder>
  <zorder>label_13</zorder>
  <zorder>NationEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>ReachEdit</zorder>
  <zorder>PlaceEdit</zorder>
  <zorder>label_7</zorder>
  <zorder>WorkNumEdit</zorder>
  <zorder>CancelButton</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>TelEdit</zorder>
  <zorder>WorkTelEdit</zorder>
  <zorder>PostEdit</zorder>
  <zorder>label_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
